scripts/bootstrap.sh

```diff
@@ -268,6 +268,15 @@ fix_jdk()
     fi
 }
 
+fix_npm_python()
+{
+    if [[ $EDX_CONFIGURATION_PUBLIC_GITHUB_ACCOUNTNAME == edx ]] ; then
+      if [[ $EDX_CONFIGURATION_PUBLIC_GITHUB_PROJECTBRANCH == *"release/ficus"* ]] ; then
+        git cherry-pick -x 075d69e6c7c5330732ec75346d02df32d087aa92
```

Comment:
Why are we not simply cherry picking this and applying this to our branch permanently vs this approach?

Reply:
this is the case when we're deploying specific ficus tags from the edx upstream (like ficus.1 and ficus.4). (ie - no microsoft repos or customizations)
we a) don't own the repositories so down-porting this fix to a ficus branch would require collaboration with edx and b) even if we do that: the fix will not be included in the ficus tags (like ficus.1 and ficus.4) which are "locked" to a moment in time before the fixes existed

Reply:
deploying microsoft repos ( like our edx-configuration ) will have the fixes by default (as a result of Microsoft/edx-configuration#59 )